helpers/kubernetes/cluster.go

```diff
@@ -393,6 +425,40 @@ func (c *Cluster) ListPods(ctx context.Context, namespace, selector string) (*v1
 	return podList, nil
 }
 
+// ListJobs returns the list of currently scheduled or running Jobs in `namespace` with the given selector
+func (c *Cluster) ListJobs(ctx context.Context, namespace, selector string) (*apibatchv1.JobList, error) {
+	listOptions := metav1.ListOptions{}
+	if len(selector) > 0 {
+		listOptions.LabelSelector = selector
+	}
+	jobList, err := c.Kubectl.BatchV1().Jobs(namespace).List(ctx, listOptions)
+	if err != nil {
+		return nil, err
+	}
+	return jobList, nil
+}
+
+func (c *Cluster) CreateJob(ctx context.Context, namespace string, job *apibatchv1.Job) error {
+	_, err := c.Kubectl.BatchV1().Jobs(namespace).Create(
+		ctx,
+		job,
+		metav1.CreateOptions{},
+	)
+	return err
+}
+
+// DeleteJob deletes the namepace
+func (c *Cluster) DeleteJob(ctx context.Context, namespace string, name string) error {
+	policy := metav1.DeletePropagationBackground
+	err := c.Kubectl.BatchV1().Jobs(namespace).Delete(ctx, name, metav1.DeleteOptions{
+		PropagationPolicy: &policy,
+	})
+	if err != nil {
+		return err
+	}
+	return nil
```

Comment:
```suggestion
	return err
```

Reply:
Heh. Missed that when fixing the deletion policy for jobs. Will fix. Can drop the `err` also, and return the result of the fun call directly.

Reply:
Done. Commit 7eb20b7e67e5b4cff825d27bb2309b3486e1cc68.